pyanaconda/core/constants.py

```diff
@@ -267,6 +267,9 @@ class SecretStatus(Enum):
 IPMI_ABORTED = 0x9          # installation finished unsuccessfully, due to some non-exn error
 IPMI_FAILED = 0xA           # installation hit an exception
 
+# Wayland socket name to use
+WAYLAND_SOCKET_NAME = "wayland-0"
```

Comment:
I explicitly didn't use `wayland-0` in #5401 because doing so makes it impossible to do testing in a nested environment.

Reply:
Hi Neal,

Thanks for taking the time to review this PR, even though it is heavily based on your work. I just changed the compositor, you figured out the difficult bits :smile: 

Could you clarify what do you mean by nested in this context? Do you mean the compositor running a nested session (i.e., `mutter --nested`)?

I don't this that mutter is using the `WAYLAND_DISPLAY` environment variable as Weston does... Or at least I wasn't able to find a way to find a way to force it to use it. In this PR `WAYLAND_SOCKET_NAME` is meant to store the default socket name rather than a custom one.

Reply:
When I mean nested, I mean when I want to pretend that there's no Wayland compositor environment while still running in a regular one. It's very useful for debugging and developing (and I did that for both anaconda initial-setup and anaconda itself). Mutter and gnome-kiosk can do this with `--wayland-display` set to the `WAYLAND_SOCKET_NAME`.

Reply:
This is part of the reason why the X display number isn't `0` as well.

Reply:
Nice, thanks for the tip! I changed the socket name in the latest version.